picard/ui/scripteditor.py

```diff
@@ -593,6 +593,13 @@ def load(self):
         if not self.loading:
             self.select_script(skip_check=True)
 
+    def reload_after_profile(self):
+        """Update after changes to profile settings.
+        """
+        config = get_config()
+        id = config.setting["selected_file_naming_script_id"]
```

Comment:
`id` is a builtin function, so it is best to avoid using it as variable name (I know, it sucks).
But `script_id` would work well here instead.

Reply:
I've done that so often you'd think I'd remember it already.  Apparently not.  :frowning: